extensions/datastores/kudu/src/main/java/org/locationtech/geowave/datastore/kudu/operations/KuduMetadataWriter.java

```diff
@@ -0,0 +1,75 @@
+package org.locationtech.geowave.datastore.kudu.operations;
+
+import org.apache.kudu.client.Insert;
+import org.apache.kudu.client.KuduException;
+import org.apache.kudu.client.KuduSession;
+import org.apache.kudu.client.PartialRow;
+import org.locationtech.geowave.core.store.entities.GeoWaveMetadata;
+import org.locationtech.geowave.core.store.operations.MetadataWriter;
+import org.locationtech.geowave.datastore.kudu.KuduMetadataRow;
+import org.locationtech.geowave.datastore.kudu.KuduMetadataRow.KuduMetadataField;
+import org.locationtech.geowave.datastore.kudu.util.KuduUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.nio.ByteBuffer;
+
+public class KuduMetadataWriter implements MetadataWriter {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(KuduMetadataWriter.class);
+  private final KuduOperations operations;
+  private final String tableName;
+  private final KuduSession session;
+
+  public KuduMetadataWriter(KuduOperations operations, String tableName) {
```

Comment:
`CassandraMetadataWriter` passes in the table name, but I think we should pass in `MetadataType` instead, for both consistency with the reader/deleter and for safety (to ensure that the table name is valid). Seems like both accumulo and hbase implementations do this